assets/javascripts/discourse/initializers/extend-for-assigns.js

```diff
@@ -927,13 +928,19 @@ export default {
 
       api.addUserSearchOption("assignableGroups");
 
+      let BulkAssignComponent = BulkAssign;
+      if (currentUser?.use_experimental_topic_bulk_actions) {
+        BulkAssignComponent = BulkActionsAssignUser;
+      }
+
```

Comment:
```suggestion
      const bulkAssignComponent = currentUser?.use_experimental_topic_bulk_actions ? BulkActionsAssignUser : BulkAssign;
```

Having the variable starting with a capital is a bit weird, makes it look like a new class :)

Reply:
oh whoops, just an oversight :)